src/System.Net.Http/src/System/Net/Http/Headers/HttpResponseHeaders.cs

```diff
@@ -160,5 +161,8 @@ internal override void AddHeaders(HttpHeaders sourceHeaders)
         }
 
         private HttpGeneralHeaders GeneralHeaders => _generalHeaders ?? (_generalHeaders = new HttpGeneralHeaders(this));
+
+        // Lazy initialization.
+        internal static HttpResponseHeaders EmptyResponseHeader => _emptyHeaders ?? (_emptyHeaders = new HttpResponseHeaders());
```

Comment:
Will this header collection be mutable? Seems like weird things could happen if someone actually adds a header to this collection.

Reply:
Yeah, I need to make this collection something like `Readonly`. Will fix.